spec/models/custom_button_spec.rb

```diff
@@ -294,9 +294,9 @@
   end
 
   describe "publish custom button event" do
-    let(:vm)              { FactoryGirl.create(:vm_vmware) }
-    let(:vm2)             { FactoryGirl.create(:vm_vmware) }
-    let(:vm3)             { FactoryGirl.create(:vm_vmware) }
+    let(:vm)             { FactoryGirl.create(:vm_vmware) }
+    let(:vm2)            { FactoryGirl.create(:vm_vmware) }
+    let(:vm3)            { FactoryGirl.create(:vm_vmware) }
```

Comment:
I like that after a few rounds of changes the sole purpose here is to undo the alignment with the `let` blocks below.  :stuck_out_tongue_closed_eyes: